engines/native/sandbox.go

```diff
@@ -19,7 +21,7 @@ type sandbox struct {
 	engine        *engine
 	context       *runtime.TaskContext
 	log           *logrus.Entry
-	workingFolder runtime.TemporaryFolder
+	workingFolder string
```

Comment:
I propose keeping `runtime.TemporaryFolder` but leaving it `nil` if we're not creating a folder...

That way we reuse the cleanup logic in `runtime`